tools/circle_plus_gen/lib/json_parser.py

```diff
@@ -22,6 +27,7 @@ def _generate_optimizer(
     # e.g. SGDOptionsT.learningRate = 0.001
     optimizer_opt = opt_type()
     for (key, value) in args.items():
+        key = to_camel_case(key)
```

Comment:
This is to accept the snake case variable.

### before

only camel case was acceptable.
```json
{
  "optimizer": {
      "type": "adam",
      "args": {
          "learningRate": 0.01,
          "beta1": 0.9,
          "beta2": 0.999,
          "epsilon": 1e-07
      }
...
} 
```

### after 

both snake case and camel cases are acceptable.

```json 
{
  "optimizer": {
      "type": "adam",
      "args": {
          "learning_rate": 0.01,
          "beta_1": 0.9,
          "beta_2": 0.999,
          "epsilon": 1e-07
      }
...
} 
```

Reply:
This is to ease writing document. 

The document referred `*.fbs` which uses snake case - https://github.com/Samsung/ONE/blob/master/runtime/libs/circle-schema/include/circle_traininfo.fbs.